modules/ROOT/examples/deployment/container/orchestration/nginx-ingress-tab-1.yaml

```diff
@@ -1,29 +1,28 @@
-externalDomain: ocis.owncloud.test
+externalDomain: ocis.kube.owncloud.test
 
-  ingress:
-    enabled: true
-    ingressClassName: nginx
-    annotations:
-      cert-manager.io/issuer: "ocis-certificate-issuer"
-    tls:
-      - hosts:
-        - ocis.owncloud.test
-        secretName: ocis-tls-certificate
+ingress:
+  enabled: true
+  ingressClassName: nginx
+  annotations:
+    cert-manager.io/issuer: 'ocis-certificate-issuer'
+  tls:
+    - hosts:
+        - ocis.kube.owncloud.test
+      secretName: ocis-tls-certificate
 
-  extraResources:
-    - |
-      apiVersion: cert-manager.io/v1
-      kind: Issuer
-      metadata:
-        name: ocis-certificate-issuer
-        namespace: ocis-namespace
```

Comment:
I have removed `namespace: ocis-namespace` entry. If that is to include then we might need to document somewhere a command to add namespace
such as `minikube kubectl -- create namespace ocis-namespace`

Reply:
Oh! I remember that I got a namespace not found message.
Please, confirm from your side